pyworkflow/plugin.py

```diff
@@ -620,12 +620,58 @@ def getKeywords(self):
 
 
 class Template:
-    def __init__(self, pluginName, tempPath, description="Description not provided"):
+    def __init__(self, pluginName, tempPath):
         self.pluginName = pluginName
         self.templateName = os.path.basename(tempPath).replace(SCIPION_JSON_TEMPLATES, "")
         self.templatePath = os.path.abspath(tempPath)
-        self.description = description
+        self.description, self.content = self.parseTemplate(self.templatePath)
 
     def getObjId(self):
         return self.pluginName + '-' + self.templateName
 
+    @staticmethod
```

Comment:
Do we expect this method to be invoked without having an instance? otherwise I'd have them not static nor classmethods.